components/layout_thread_2020/dom_wrapper.rs

```diff
@@ -484,7 +484,7 @@ impl<'le> TElement for ServoLayoutElement<'le> {
         let node = self.as_node();
         let document = node.owner_doc();
         context.animations.get_animation_declarations(
-            &AnimationSetKey(node.opaque()),
+            &AnimationSetKey(node.opaque(), None /* pseudo_element */),
```

Comment:
Perhaps we should have a `AnimationSetKey::non_pseudo` constructor instead.

Reply:
This is a great idea. I've added three constructors `new`, `new_for_non_pseudo`, and `new_for_pseudo`.